### ISSUE TITLE
tooltip for graphs

### ISSUE THREAD

**Issue by tivvit** [gdg-garage/space-tycoon#104]

**Comment by malytomas** [comment]
do you mean tooltips to different types of graphs - to explain what it is.

or do you mean tooltips to individual "points" on the lines? (this might be difficult due to performance considerations)

**Comment by tivvit** [comment]
![image](https://user-images.githubusercontent.com/3392931/186778718-166f6ce0-7e92-4f6e-b26f-9bb4dc2c9425.png)

**Comment by malytomas** [comment]
yeah i thought so. this would require to first implement some additional filtering of the points. the browser can barely handle it as it is now (and there already is some filtering...)
